app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -290,3 +290,7 @@ div.form-group .pficon-info{
 a.btn-info span{
   padding-bottom: 4px;
 }
+
+.btn-toolbar .btn.btn-docs {
```

Comment:
yeah but that's not the selector in line 146.
If this is the only comment you can change it at merge - or I can re-send it